src/+matlode/+rk/+erk/ERK.m

```diff
@@ -78,40 +50,51 @@
             tCur = tspan(1);
             tNext = tCur;
             ti = 2;
-            hmax = opts.MaxStep;
+            
+            tdir = sign(tspan(end) - tspan(1));
+            hmax = opts.MaxStep * tdir;
+            hmin = 64 * eps(tspan(1)) * tdir;
```

Comment:
I really like this except for one small giant fact, eps(0) = realmin. Realistically, we can do

`tdir*eps(tspan(end) - tspan(1))`, which should hopefully give us something better? 

Around zero everything breaks. Floats are not that great. There are representations that would not have this problem, but here we are...

Maybe something like,

`tdir*max(eps(tspan(end)), eps(tspan(1))`,

or something like that.

Reply:
hmm, interestingly matlab does not address this, do we need to?